### DISCUSSION TITLE
i won

### DISCUSSION THREAD

**Discussion by serifonium** [aeolus-1/triangleGame.github.io#53] [Show and tell]
i won
<img width="1440" alt="Screen Shot 2022-04-22 at 9 25 31 am" src="https://user-images.githubusercontent.com/104041484/164566980-43503e20-b001-4b31-af42-2b9d4f5f51a5.png">

**Comment by nekoify** [comment]
legit

**Comment by Quantaviousdingleberry420** [comment]
yeah actual legit tho

**Comment by aeolus-1** [comment]
someone set up speedrun.com plz
